WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java

```diff
@@ -820,9 +820,13 @@ public static void scrollIntoView(Integer scrollableContainerID, ViewInteraction
     }
 
     public static void dismissJetpackAdIfPresent() {
+        String jpAdText = "Stats, Reader, Notifications, and other features are powered by Jetpack.";
```

Comment:
nitpick, I think writing it as Jetpack instead of jp makes it easier to read, the word Jetpack is already being used in the function name and comment
```suggestion
        String jetpackAdText = "Stats, Reader, Notifications, and other features are powered by Jetpack.";
```

Reply:
Good point, thanks! Renamed in ba7d064a70ffd75887c1c0530b5f3d183026c4a9.